src/UI/Seller/src/app/buyers/components/locations/buyer-location-catalogs/buyer-location-catalogs.component.ts

```diff
@@ -12,14 +12,12 @@ import { CatalogsTempService } from '@app-seller/shared/services/middleware-api/
   styleUrls: ['./buyer-location-catalogs.component.scss'],
 })
 export class BuyerLocationCatalogs {
-  buyerID = ''
+  buyerID = this.router.routerState.snapshot.url.split('/')[2]
```

Comment:
same router comment here, It doesn't seem like we can guarantee that element being there at this point in the code.